block/cardano-sl-block.cabal

```diff
@@ -154,3 +156,16 @@ library
 
   build-tools: cpphs >= 1.19
   ghc-options: -pgmP cpphs -optP --cpp
+
+Benchmark bench-block
+    type: exitcode-stdio-1.0
+    main-is: Block.hs
+    hs-source-dirs: bench
+    ghc-options: -O2
+    build-depends:   criterion >= 1.3.0.0
+                   , cardano-sl-crypto
+                   , cardano-sl-core
+                   , cardano-sl-txp
+                   , cardano-sl-update
+                   , cardano-sl-ssc
+                   , cardano-sl-dlg
```

Comment:
It's called `cardano-sl-delegation`. How do you build the project?

Reply:
I never built the benchmark via cabal/stack, I built it directly with ghc. There are other missing build-depends. Will fix this.

Reply:
Most of the devs use `ln -s ./scripts/build/cardano-sl.sh b` and run `./b infra` or something. The script is really good, you should check it if you missed it somehow.